IPython/testing/iptest.py

```diff
@@ -57,6 +57,7 @@
 warnings.filterwarnings('error', message='.*disable_gui.*', category=DeprecationWarning, module='.*')
 
 warnings.filterwarnings('error', message='.*ExceptionColors global is deprecated.*', category=DeprecationWarning, module='.*')
+warnings.filterwarnings('error', message='.*IPython.core.display.*', category=DeprecationWarning, module='.*')
```

Comment:
and then we don't have to add more filter warnings :)

Reply:
I want us to avoid internal imports,as if we do import a single time then we silence the warnings.